app/views/layouts/application.html.haml

```diff
@@ -7,5 +7,29 @@
     = stylesheet_link_tag    'application', media: 'all'
     = javascript_include_tag 'application'
     = csrf_meta_tags
+
   %body
+    %nav
+      .nav-wrapper
+        %a{ href: '#', class: 'brand-logo'} Mycademy
+        %ul{ id: 'nav-mobile', class: 'right hide-on-med-and-down'}
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should end with one space before the closing brace